docs/classes/Wallet.html

```diff
@@ -341,14 +341,18 @@ <h5><code class="tsd-tag ts-flagOptional">Optional</code> master<wbr/>Address<sp
 <ul>
-<li>Defined in <a href="https://github.com/XRPLF/xrpl.js/blob/8921b6ac/packages/xrpl/src/Wallet/index.ts#L185">packages/xrpl/src/Wallet/index.ts:185</a></li></ul></aside></li></ul></section>
+<li>Defined in <a href="https://github.com/XRPLF/xrpl.js/blob/9ea83bab/packages/xrpl/src/Wallet/index.ts#L174">packages/xrpl/src/Wallet/index.ts:174</a></li></ul></aside></li></ul></section>
 <section class="tsd-panel tsd-member tsd-kind-method tsd-parent-kind-class"><a id="fromMnemonic" class="tsd-anchor"></a>
 <h3 class="tsd-anchor-link"><code class="tsd-tag ts-flagStatic">Static</code> <span>from<wbr/>Mnemonic</span><a href="#fromMnemonic" aria-label="Permalink" class="tsd-anchor-icon"><svg class="icon icon-tabler icon-tabler-link" viewBox="0 0 24 24" stroke-width="2" stroke="currentColor" fill="none" stroke-linecap="round" stroke-linejoin="round"><use href="#icon-anchor-a"></use><use href="#icon-anchor-b"></use><use href="#icon-anchor-c"></use></svg></a></h3>
 <ul class="tsd-signatures tsd-kind-method tsd-parent-kind-class">
 <li class="tsd-signature tsd-anchor-link" id="fromMnemonic.fromMnemonic-1">from<wbr/>Mnemonic<span class="tsd-signature-symbol">(</span>mnemonic<span class="tsd-signature-symbol">: </span><span class="tsd-signature-type">string</span>, opts<span class="tsd-signature-symbol">?: </span><span class="tsd-signature-symbol">{ </span>algorithm<span class="tsd-signature-symbol">?: </span><span class="tsd-signature-type">ECDSA</span><span class="tsd-signature-symbol">; </span>derivationPath<span class="tsd-signature-symbol">?: </span><span class="tsd-signature-type">string</span><span class="tsd-signature-symbol">; </span>masterAddress<span class="tsd-signature-symbol">?: </span><span class="tsd-signature-type">string</span><span class="tsd-signature-symbol">; </span>mnemonicEncoding<span class="tsd-signature-symbol">?: </span><span class="tsd-signature-type">&quot;bip39&quot;</span><span class="tsd-signature-symbol"> | </span><span class="tsd-signature-type">&quot;rfc1751&quot;</span><span class="tsd-signature-symbol"> }</span><span class="tsd-signature-symbol">)</span><span class="tsd-signature-symbol">: </span><a href="Wallet.html" class="tsd-signature-type" data-tsd-kind="Class">Wallet</a><a href="#fromMnemonic.fromMnemonic-1" aria-label="Permalink" class="tsd-anchor-icon"><svg class="icon icon-tabler icon-tabler-link" viewBox="0 0 24 24" stroke-width="2" stroke="currentColor" fill="none" stroke-linecap="round" stroke-linejoin="round"><use href="#icon-anchor-a"></use><use href="#icon-anchor-b"></use><use href="#icon-anchor-c"></use></svg></a></li>
 <li class="tsd-description">
 <div class="tsd-comment tsd-typography"><p>Derives a wallet from a bip39 or RFC1751 mnemonic (Defaults to bip39).</p>
 
+<h3>Deprecated</h3><p>since version 2.6.1.
+Will be deleted in version 3.0.0.
+Currently this implementation does not provide better reliability/Security, please use other methods to generate wallet.</p>
```

Comment:
Are the generated docs necessary? (I think they'll be made when we do a release anyway, so may be worth removing changes to this file)

If they're important to keep - this is still showing the old message.

Reply:
good catch - I think we need to keep it because I manually updated the generate wallet examples, but the deprecation message should be updated now